dev-python/python-gnupg/python-gnupg-0.4.3.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6,3_7} pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="Python wrapper for GNU Privacy Guard"
+HOMEPAGE="https://pythonhosted.org/python-gnupg/ https://github.com/vsajip/python-gnupg/"
```

Comment:
The GitHub thing seems to be a mirror of bitbucket repo.

Reply:
FYI: I only did a small version bump. So I can't say, why in the past all your mentionted changes have been done. I also never have maintained this package. Anyway, I've removed the URL. Should I do also for `metadata.xml`?